snap-wrappers/bin/client-common.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
-
-export SNAP_REAL_HOME=$( getent passwd $USER | cut -d: -f6 )
```

Comment:
Apparently, yes :sweat_smile:
So, there are two things here:
- the `link_autostart` thing, which is not needed anymore because of the new autostart mechanism from the GUI.
- the `SNAP_REAL_HOME` var, which to be honest I didn't look enough into. Apparently it is used in one place in the code (for getting the home directory in the `launch` command). So I guess the file should still stay for that variable. But what I find interesting is that we are setting the variable to the value it already has by default. So even without setting it explicitly, `SNAP_REAL_HOME` already points to the actual home directory of the current user. So I wonder why it was needed in the first place.

Reply:
I am not sure right now, but this was added in this commit: https://github.com/canonical/multipass/commit/40577608669c3f61c23311e20c0b0bb9dd35726f from https://github.com/canonical/multipass/pull/1074 I believe

Reply:
I've put it back just to be safe.